recipes/omr1/configs/full-tir-stage-0.yaml

```
@@ -0,0 +1,57 @@
+cluster: oci
```

```suggestion
cluster: slurm
```
Please also update other configs not included in your PR